src/Curation/Curation.router.ts

```diff
@@ -181,7 +193,26 @@ export class CurationRouter extends Router {
   insertItemCuration = async (req: AuthRequest) => {
     const itemId = server.extractFromReq(req, 'id')
     const ethAddress = req.auth.ethAddress
-    return this.insertCuration(itemId, ethAddress, CurationType.ITEM)
+    const itemCuration = await this.insertCuration(
+      itemId,
+      ethAddress,
+      CurationType.ITEM
+    )
+
+    const item = await Item.findOne(itemId)
+    const collectionCuration = await CollectionCuration.findOne(
+      item.collection_id
+    )
+
+    if (!collectionCuration) {
+      await this.insertCuration(
+        item.collection_id,
+        ethAddress,
+        CurationType.COLLECTION
+      )
```

Comment:
Maybe we could create a trigger in the DB to do this, but it's just something to think about.

Reply:
Agree it could be much better. Triggers sometime make me uncomfortable as they're "hidden" logic that's super hard to debug in production tho. But we could check it out